spec/views/events/schedule.html.erb_spec.rb

```diff
@@ -27,9 +27,28 @@
       expect(rendered).to have_selector("input[value='31.12.2017']")
     end
 
-    it 'has edit button for gameday dates' do
-      render
-      expect(rendered).to have_selector(:link_or_button, t('events.schedule.edit_date'))
+    describe 'authorization for edit date button' do
+      before(:each) do
+        @user = FactoryBot.create(:user)
+        sign_in @user
+      end
+
+      it 'has edit button for gameday dates for signed in users' do
```

Comment:
Consider using the following as description: `has edit button for gameday dates for signed in users with rights as organizer of the league`